server/service/individualServices/PrepareApplicationData.js

```diff
@@ -0,0 +1,89 @@
+const fileSystem = require('fs')
+/*
+* Add new applications if there is no instance available for this application + release-number combination
+* Update approval status if there is an instance available for this application + release-number combination
+*/
+exports.addAndUpdateApplicationData = async (filePath, applicationData) => {
+    return new Promise(async function (resolve, reject) {
+        try{
+            fileSystem.writeFileSync(filePath, JSON.stringify(applicationData), (errWritingIntoFile) => {
```

Comment:
The application data file we are trying to access might contain username ,password and permission like READ_WRITE , READ_ONLY etc. Information is present in file profile .While reading or writing to file we have get the information from file profile and access accordingly

Reply:
let's create a new issue for it and will cover this there.